staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2156,6 +2156,29 @@ type VolumeMount struct {
 	// Defaults to false.
 	// +optional
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,2,opt,name=readOnly"`
+	// RecursiveReadOnly specifies whether read-only mounts should be handled
+	// recursively.
+	//
+	// If ReadOnly is false, this field has no meaning and must be unspecified.
+	//
+	// If ReadOnly is true, and this field is set to Disabled, the mount is not made
+	// recursively read-only.  If this field is set to IfPossible, the mount is made
+	// recursively read-only, if it is supported by the container runtime.  If this
+	// field is set to Enabled, the mount is made recursively read-only if it is
+	// supported by the container runtime, otherwise the pod will not be started and
+	// an error will be generated to indicate the reason.
+	//
+	// If this field is set to IfPossible or Enabled, MountPropagation must be set to
+	// None (or be unspecified, which defaults to None).
+	//
+	// If this field is not specified on a Pod it defaults to Disabled for backwards
```

Comment:
This is tricky.  We *want* to add static defaulting logic, but we have other issues around Kubelet that have given us a moratorium on new fields with default values.  So we *can't* add a static default here, which means the implementation code has to handle `nil` and `"Disabled"` as equivalent.

Reply:
Reworded to ` If this field is not specified, it is treated as an equivalent of Disabled.`